Core/Core/ViewModifiers/iOS16HideListScrollContentBackground.swift

```diff
@@ -1,40 +0,0 @@
-//
-// This file is part of Canvas.
-// Copyright (C) 2022-present  Instructure, Inc.
-//
-// This program is free software: you can redistribute it and/or modify
-// it under the terms of the GNU Affero General Public License as
-// published by the Free Software Foundation, either version 3 of the
-// License, or (at your option) any later version.
-//
-// This program is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU Affero General Public License for more details.
-//
-// You should have received a copy of the GNU Affero General Public License
-// along with this program.  If not, see <https://www.gnu.org/licenses/>.
-//
-
-import SwiftUI
-
-@available(iOS, introduced: 16.0, obsoleted: 16.1)
```

Comment:
I think this one was incorrectly labeled as obsolete on 16.1 because it's needed even on 17.5. Otherwise there's a leading inset.

Reply:
iOS16HideListScrollContentBackground's scrollContentBackground is not hidden by default on iOS 16.0. Since our minimum supported version is 16.0, we'd need to add it to `List` components anyway, so I'm not sure if we benefit anything from having an extra view modifier versus writing out `scrollContentBackground(.hidden)`.